### ISSUE TITLE
Feature request: show block age when importing / blocking

### ISSUE THREAD

**Issue by karalabe** [filecoin-project/lotus#3307]
In Geth we've found it super useful to show the age of a block being imported (e.g. `Imported block #N, age=1d2h`). It helps you see at a glance how far out you are with sync.

Same thing when you start the miner and it's waiting for the daemon to sync up: `State: header sync      Height: 4185` is useful, but says absolutely zero about how much time it will still take. I could of course look the block up in an explorer, but that's weird given that I already have the information locally. Printing `State: header sync      Height: 4185 Age: 2d1h` would really help gauge the time you need to wait.

Edit: For the second use case, it might even make sense to try and guess an ETA based on the import time of the last N blocks and the total chain length we're seemingly pulling.

**Comment by f8-ptrk** [comment]
i think lotus is doing this as by now - 1.9.0

#antidisestablishatarianism

**Comment by frrist** [comment]
This issues has been addressed by #3330 and its effect can be seen in the logs of the lotus daemon:
```
2021-07-20T11:43:30.122-0700    INFO    chain   chain/sync_manager.go:232       selected sync target: [bafy2bzacea52dlb6bqpvlgw5jnbirrpjihyigubx3swbv6wcgiwvleutbxfy2]
2021-07-20T11:43:30.122-0700    INFO    chain   chain/sync_manager.go:314       worker 481 syncing in [bafy2bzacea52dlb6bqpvlgw5jnbirrpjihyigubx3swbv6wcgiwvleutbxfy2]
2021-07-20T11:43:30.134-0700    INFO    chain   chain/sync_manager.go:232       selected sync target: [bafy2bzaceckmxnstafcdi7d4ovn775427qt73l64yz32okgre3gry4ux24fpq]
2021-07-20T11:43:30.147-0700    INFO    chain   chain/sync.go:700       block validation        {"took": 0.021874116, "height": "91527", "age": 0.147369178}
2021-07-20T11:43:30.150-0700    INFO    chainstore      store/store.go:646      New heaviest tipset! [bafy2bzacea52dlb6bqpvlgw5jnbirrpjihyigubx3swbv6wcgiwvleutbxfy2 bafy2bzacedna6i3nqt7uwwbvwgojbfynmpnriesdae46mgjtapjmefbcm24xw] (height=91527)
2021-07-20T11:43:30.153-0700    INFO    chain   chain/sync_manager.go:322       worker 481 done; took 30.245244ms
```
